netci.groovy

```diff
@@ -981,6 +986,11 @@ def static addNonPRTriggers(def job, def branch, def isPR, def architecture, def
         case 'tailcallstress':
         case 'jitsse2only':
         case 'jitnosimd':
+        case 'jitnox86hwintrinsic':
+        case 'jitx86hwintrinsic':
+        case 'jitx86hwintrinsicsseencoding':
```

Comment:
Perhaps to make it shorter: `jitx86hwintrinsicssee`, in other constants there are only combinations of `no` and ISA shortcut `avx`, this would be the same after change here

Reply:
> Perhaps to make it shorter: jitx86hwintrinsicssee, in other constants there are only combinations of no and ISA shortcut avx

Yes, `noavx` is also okay.

Reply:
Yes, and even we should have no name collisions with other intrinsics as AFAIR Intel is protecting them as trademarks.